tox.ini

```diff
@@ -10,3 +10,6 @@ envlist = py26, py27, py34, pypy
 commands = py.test
 deps =
     pytest
+    fontTools
+    robofab
```

Comment:
I know, the "dependency hell" is a pain :)

If defcon/ufo3 still relies on the deprecated robofab/ufo3k branch, it should be fixed in defcon, by importing the standalone ufoLib package.

I believe @adrientetar was working on merging some of his Trufont patches to upstream defcon.

I suggest you use a `requirements.txt` file to use with `pip install -r requirements.txt` containing the `git+https:...` urls of the required fontTools and ufoLib HEAD packages.